marklogic-data-hub/src/main/java/com/marklogic/hub/StepManager.java

```diff
@@ -0,0 +1,81 @@
+package com.marklogic.hub;
```

Comment:
Can you toss this into the com.marklogic.hub.impl package? I think there was an intent for com.marklogic.hub to define public APIs that would be well-documented for use, but I don't think that has materialized. Because we're not making that a priority, I want to avoid the perception of creating public APIs, and I think we can thus "hide" stuff in the impl package to avoid this.

Reply:
done